.github/pull_request_template.md

```diff
@@ -12,7 +12,15 @@ see https://regro.github.io/rever-docs/news.html for details on how to add news
 -->
 
 Checklist
-* [ ] Added a ``news`` entry
+* [ ] All new code is appropriately documented (user-facing code _must_ have complete docstrings).
+* [ ] Added a ``news`` entry, or the changes are not user-facing.
+* [ ] Ran pre-commit by making a comment with `pre-commit.ci autofix` before requesting review.
+
+Manual Tests: these are slow so don't need to be run every commit, only before merging. 
```

Comment:
I think it might be good to say "if it's relevant", a bit like you say for packaging tests below.

Reply:
done!